File: app/src/main/java/mb00/android/codehub/ui/search/viewmodel/SearchReposViewModel.java
package mb00.android.codehub.ui.search.viewmodel;

import mb00.android.codehub.ui.search.interactor.SearchInteractor;
import mb00.android.codehub.ui.search.router.SearchRouter;


public class SearchReposViewModel extends SearchViewModel {

    public SearchReposViewModel(SearchInteractor interactor, SearchRouter router) {
        super(interactor, router);
    }

}

File: app/src/main/java/mb00/android/codehub/ui/search/viewmodel/SearchReposViewModel.kt
package mb00.android.codehub.ui.search.viewmodel

import io.reactivex.Single
import mb00.android.codehub.api.model.Repo
import mb00.android.codehub.ui.search.interactor.SearchInteractor
import mb00.android.codehub.ui.search.router.SearchRouter


class SearchReposViewModel(private val interactor: SearchInteractor, private val router: SearchRouter)
    : SearchViewModel(interactor, router) {

    /*fun searchRepos(header: String, user: String): Single<List<Repo>> {
        interactor.search()
    }*/

}
